apps/coscrad-frontend/src/utils/generic-components/presenters/tables/generic-index-table-presenter/index-table.tsx

```diff
@@ -59,14 +60,18 @@ export const IndexTable = <T extends IBaseViewModel>({
     const [searchValue, setSearchValue] = useState('');
 
     // SEARCH LOGIC
-    const filteredTableData =
-        searchValue.length === 0
-            ? tableData
-            : tableData.filter((row) =>
-                  filterableProperties.some((propertyKey) =>
-                      String(row[propertyKey]).toLowerCase().includes(searchValue.toLowerCase())
-                  )
-              );
+
+    const [selectedFilterProperty, setSelectedFilterProperty] = useState(('allColumns' || filterableProperties[0]));
```

Comment:
'allColumns' || filterableProperties[0] will always return 'allColumns' since 'allColumns' the non-empty literal string value will always be truthy.